convert2rhel/main.py

```diff
@@ -203,6 +203,7 @@ def main_locked(process_phase):
         # parse and act upon it.
         if pre_conversion_results:
             actions.report.summary_as_json(pre_conversion_results)
+            actions.report.report_to_txt(pre_conversion_results)
```

Comment:
Can we change the name of the function to follow the same as the above? 

```suggestion
            actions.report.summary_as_json(pre_conversion_results)
```

Mainly to keep it consistent with what we have and pass the notion that we are still talking about a summary, but this time, in textual form.